pages.it/common/ffprobe.md

```diff
@@ -0,0 +1,24 @@
+# ffprobe
+
+> Analizzatore di flussi multimediali.
+> Maggiori informazioni: <https://ffmpeg.org/ffprobe.html>.
+
+- Visualizza tutte le informazioni disponibili sui flussi di un file multimediale (audio, video, immagini, etc):
+
+`ffprobe -v error -show_entries {{file.mp4}}`
+
+- Visualizza la durata del contenuto:
+
+`ffprobe -v error -show_entries format=duration -of default=noprint_wrappers=1:nokey=1 {{file.mp4}}`
+
+- Visualizza la frequenza dei fotogrammi di un video:
+
+`ffprobe -v error -select_streams v:0 -show_entries stream=avg_frame_rate -of default=noprint_wrappers=1:nokey=1 {{video.mp4}}`
+
+- Visualizza la larghezza o l'altezza di un video:
+
+`ffprobe -v error -select_streams v:0 -show_entries stream={{width|height}} -of default=noprint_wrappers=1:nokey=1 {{video.mp4}}`
+
+- Visualizza il bit-rate medio di un video:
+
+`ffprobe -v error -select_streams v:0 -show_entries stream=bit_rate -of default=noprint_wrappers=1:nokey=1 {{input.mp4}}`
```

Comment:
I `input` is not Italian. Please translate the arguments for get this easier for non-English speakers, you can use:
```suggestion
`ffprobe -v error -select_streams v:0 -show_entries stream=bit_rate -of default=noprint_wrappers=1:nokey=1 {{video.mp4}}`
```
here